packages/react/src/components/Tag/Tag.js

```diff
@@ -85,6 +87,11 @@ Tag.propTypes = {
    */
   filter: PropTypes.bool,
 
+  /**
+   * Determine if <Tag> is a 18px high
+   */
+  short: PropTypes.bool,
```

Comment:
Wondering if we can make `size: PropTypes.oneOf(['regular', 'short'])`. We used to have `small: PropTypes.bool` in `<Button>` but we had to deprecate that prop due to additional size coming up: https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/Button/Button.js#L131-L144